app/locales/de/messages.po

```diff
@@ -571,11 +571,15 @@ msgstr "Tab Layout"
 
 #: app/configurator/components/field-i18n.ts
 msgid "controls.layout.tall"
-msgstr ""
+msgstr "Hoch"
+
+#: app/configurator/components/field-i18n.ts
+msgid "controls.layout.tiles"
+msgstr "Kacheln"
 
 #: app/configurator/components/field-i18n.ts
 msgid "controls.layout.vertical"
-msgstr ""
+msgstr "Hoch"
```

Comment:
Not 100% sure what we should put here, but for other languages translations for Tall and Vertical layout were different (here they are the same) 👀

Reply:
Fixed